code/main/common/constants.brs

```diff
@@ -57,12 +57,6 @@ function _adb_InternalConstants() as object
                 SESSION_END: "media.sessionEnd",
                 SESSION_COMPLETE: "media.sessionComplete"
             }
-            ''' TODO remaining constants
-            EVENT_TYPES: ["media.sessionStart", "media.play", "media.ping", "media.bitrateChange", "media.bufferStart", "media.pauseStart", "media.adBreakStart", "media.adStart", "media.adComplete", "media.adSkip", "media.adBreakComplete", "media.chapterStart", "media.chapterComplete", "media.chapterSkip", "media.error", "media.sessionEnd", "media.sessionComplete", "media.statesUpdate"],
-            SESSION_START_EVENT_TYPE: "media.sessionStart",
-            SESSION_END_EVENT_TYPE: "media.sessionEnd",
-            PING_EVENT_TYPE: "media.ping",
-            SESSION_START_EDGE_REQUEST_PATH: "/ee/va/v1/sessionStart",
```

Comment:
Was this constant removed intentionally?

Reply:
Yes, have updated the usages.